python/servo/command_base.py

```diff
@@ -814,7 +817,7 @@ def run_cargo_build_like_command(
             env['RUSTFLAGS'] = env.get('RUSTFLAGS', "") + " --cfg=crown"
 
         if "-p" not in cargo_args:  # We're building specific package, that may not have features
-            features = list(self.features)
+            features += list(self.features)
```

Comment:
I am pretty sure this will bite us somewhere, but it will be removed in followup anyway.